Vitalsign mii fixes

patrick-werner opened the pull request:
# Contributor Pull Request

## Description
Konsolidierung der Profile, die der MII entstammen, zur Harmonisierung sowohl mit den Core-Profilen, als auch mit den "Alten" ISiK-Vitalparameter Profilen.

## Motivation and Context
Solves PTDATA-1592

## How has this been tested?
first review

- review pending -> wim Zuge der Kommentierung für die Stufe 5 (u.a. durch MII)

## Snippets or Screenshots (if necessary):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x]  Potentially breaking fix that would cause existing functionality to not work as expected)

### For reviewers
You migh use this: https://conventionalcomments.org/

patrick-werner committed:
fix: source of invariant was wrong

patrick-werner committed:
fix: removed VS for UCUM units, replaced usages with pattern on valueQuantity.code

patrick-werner committed:
fix: added two missing Concepts used in profiles to VS

patrick-werner committed:
fix: replaced vs binding with pattern for ucum

patrick-werner committed:
fix: made fhir-core slice mandatory to remove validation issues/fulfill the FHIR spec

patrick-werner committed:
added fsh output

patrick-werner committed:
added changelog entry

patrick-werner committed:
added new pipeline

f-peverali commented on Resources/input/fsh/mii-backport/SD_MII_ICU_Sauerstoffsaettigung_Im_Blut_Postduktal_Durch_Pulsoxymetrie.fsh:
```diff
@@ -10,7 +10,7 @@ Title: "SD MII ICU Sauerstoffsaettigung Im Blut Postduktal Durch Pulsoxymetrie"
   * coding[IEEE-11073] 1..1
   * coding[IEEE-11073] = $IEEE11073#160300
-  * coding contains loinc-fhir-core 0..* MS
+  * coding contains loinc-fhir-core 1..* MS
```
question: cardinallity Änderung im TC?
todo/issue: keine Kard. 1.. ohne motivierenden Comment bitte

patrick-werner committed:
Kommentar für Kard. Aenderung hinzugefügt, falsche fhir-core slices entfernt

patrick-werner committed:
fix: der entfernte loinc category code ist der panel-code und für die Verwendung auf .code einer Abbildung eines Panels gedacht

patrick-werner committed:
fix: magic loinc code kann max einmal auftreten

patrick-werner committed:
updated ReleaseNotes.md

patrick-werner committed:
updated ReleaseNotes.md

patrick-werner committed:
moved system pattern into SD_MII_ICU_Monitoring_Und_Vitaldaten

patrick-werner committed:
feat: added ISiK-KoerperkernTemperatur Profile
fix: integrated mii profiles into Vitalparameter Module, KoerperKernTemp Profiles of MII are now derived from ISiK-KoerperkernTemperatur
fix: fixed several validation errors

patrick-werner committed:
fix: added missing meta rulesets inserts

patrick-werner committed:
fix: add binding to specific loinc codes of generic temperature

patrick-werner committed:
fix: fixed SCT Code in ISiK-KoerperkernTemperatur example

f-peverali commented on Resources/input/fsh/mii-backport/SD_MII_ICU_Herzzeitvolumen.fsh:
```diff
@@ -10,17 +10,18 @@ Title: "SD MII ICU Herzzeitvolumen"
+* valueQuantity 
   * ^patternQuantity.unit = "liter per minute"
   * unit 1..
+  * code = #L/min
```
question: hier scheint das Einheit nicht aus einem Core-Profil zu stammen. D.h. die Einheit "deriviert", um konsistent mit anderen Profilen zu bleiben.

f-peverali commented on Resources/input/fsh/ISiK-KoerperkernTemperatur.fsh:
```diff
@@ -0,0 +1,36 @@
+Profile: ISiKKoerperkerntemperatur
+Parent: ISiKKoerpertemperatur
+Id: ISiKKoerperkerntemperatur
+Description: """Dieses Profil spezifiziert die Minimalanforderungen für die Bereitstellung von Informationen über die Körperkerntemperatur eines Patienten im Rahmen der interoperablen Kommunikation gemäß den Vorgaben der ISiK (Interoperable Schnittstelle im Krankenhaus).  
```
```suggestion
Description: """Dieses Profil spezifiziert die Minimalanforderungen für die Bereitstellung von Informationen über die Körperkerntemperatur eines Patienten im Rahmen der interoperablen Kommunikation gemäß den ISiK Vorgaben.  
```

f-peverali committed:
Update Resources/input/fsh/ISiK-KoerperkernTemperatur.fsh

f-peverali committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

f-peverali commented on Resources/input/fsh/ISiK-VitalparameterValueSets.fsh:
praise: displays passen

f-peverali reviewed (approved):
comment: wir mergen diesen PR, allerdings steht eine erneute QS durch MII noch aus

patrick-werner committed:
Merge remote-tracking branch 'origin/TC_4.0.2' into vitalsign-mii-fixes

# Conflicts:
#	ImplementationGuide/markdown/ReleaseNotes.md
#	Resources/fsh-generated/fsh-index.json
#	Resources/fsh-generated/fsh-index.txt
#	Resources/fsh-generated/resources/ValueSet-vs-mii-icu-unit-equivalent-ucum-beats-per-minute.json
#	Resources/fsh-generated/resources/ValueSet-vs-mii-icu-unit-equivalent-ucum-dyn-s-cm5-m2.json
#	Resources/fsh-generated/resources/ValueSet-vs-mii-icu-unit-equivalent-ucum-l-per-minute-and-squaremeter.json
#	Resources/fsh-generated/resources/ValueSet-vs-mii-icu-unit-equivalent-ucum-l-per-minute.json
#	Resources/input/fsh/rulesets/ruleset.fsh

patrick-werner committed:
resolve merge requests
